docs/releases/migrating-v5-v6.md

```diff
@@ -528,5 +528,5 @@ In Auspice v2, all values are now displayed exactly as they arrive, allowing use
 
 Don't forget to also change them in any custom lat-long and/or coloring files you are using. We've also become stricter about the format of the files that pass in color and lat-long information. Previously, it didn't matter if columns were separated by spaces or tabs - now, they must be separated by tabs.
 
-You can find out more about how to add [custom coloring](/faq/colors) and [lat-long](/faq/lat_longs) values.
+You can find out more about how to add [custom coloring](https://docs.nextstrain.org/en/latest/augur/docs/faq/colors.html) and [lat-long](https://docs.nextstrain.org/en/latest/augur/docs/faq/lat_longs.html) values.

```

Comment:
If / when we merge nextstrain/docs.nextstrain.org#36, this change will be necessary:
```suggestion
You can find out more about how to add [custom coloring](https://docs.nextstrain.org/en/latest/guides/bioinformatics/colors.html) and [lat-long](https://docs.nextstrain.org/en/latest/guides/bioinformatics/lat_longs.html) values.

```